scripts/latest_template_tester_report.py

```diff
@@ -24,7 +24,7 @@ def parse_cmd_line():
     parser = argparse.ArgumentParser(argument_default=None)
     parser.add_argument("--tracketbot-url", dest="trackerbot_url",
                         help="tracker bot url to make api call",
-                        default='http://10.16.4.32/trackerbot/api')
+                        default='http://http://infra-trackerbot2.cfme2.lab.eng.rdu2.redhat.com/api')
```

Comment:
REQUIRED: typo
and I would extract it from some yaml if it is possible for sure

Reply:
reworked how this is handled a bit to achieve this.